src/org/testKitGen/TestInfoParser.java

```diff
@@ -112,6 +112,19 @@ public TestInfo parse() {
 			}
 		}
 
+		String rerun = getImmediateChildContent(testEle, "rerun");
+		if (rerun != null) {
+			if (rerun.equals("true")) {
+				ti.setRerun(true);
+			} else if (rerun.equals("false")) {
+				ti.setRerun(false);
+			} else {
+				System.err.println("Error: Please provide a valid rerun parameter in test " + ti.getTestCaseName() + ". The valid string is [true|false].");
```

Comment:
Do we not have as logger for this?

Reply:
No, we don't have logger for now. It can be an improvement to add something like log4j in the future.

Reply:
For the record, loggers are extra dependencies that we have purposefully tried to keep to the very minimum for AQAvit work, as there is overhead with downloading/maintaining/updating each new dependency added.

Reply:
At some point dumping to console only is going to restrict debugging, but that's a problem for another day :-)